applications/ShapeOptimizationApplication/custom_utilities/optimization_utilities.h

```diff
@@ -151,16 +148,20 @@ class OptimizationUtilities
             for (ModelPart::NodeIterator node_i = mrDesignSurface.NodesBegin(); node_i != mrDesignSurface.NodesEnd(); ++node_i)
             {
                 array_3d design_update = step_size * ( node_i->FastGetSolutionStepValue(SEARCH_DIRECTION) );
-                noalias(node_i->FastGetSolutionStepValue(DESIGN_UPDATE)) = design_update;
-
-                // Sum design updates to obtain control point position
-                noalias(node_i->FastGetSolutionStepValue(DESIGN_CHANGE_ABSOLUTE)) += design_update;
+                noalias(node_i->FastGetSolutionStepValue(CONTROL_POINT_UPDATE)) = design_update;
             }
         }
 
         KRATOS_CATCH("");
     }
 
+    // --------------------------------------------------------------------------
+    void update_control_point_position_by_input_variable( const Variable<array_3d> &rNodalVariable )
```

Comment:
control_point_POSITION is not the right term in my opinion, as its just the relative change from the beginning of the opt. where the position is unknown